packages/ilp-protocol-stream/src/stream.ts

```diff
@@ -696,3 +692,13 @@ export class DataAndMoneyStream extends Duplex {
     }
   }
 }
+
+// We set this outside the class definition due to typing issues
+// TS2611: 'closed' is defined as a property in class 'Duplex', but is overridden here in 'DataAndMoneyStream' as an accessor.
+Object.defineProperty(DataAndMoneyStream.prototype, 'closed', {
+  get(this: DataAndMoneyStream) {
```

Comment:
Does Duplex itself ever /set/ closed ? 
The subclass covers all those cases?

Reply:
Node uses two internal objects `_readableState` and `_writableState` which each have a closed property. `Readable#closed` and `Writable#closed` are accessors for those, e.g.:

https://github.com/nodejs/node/blob/70d81bb8f72faeee3ed1163d820207fc6f57cbd1/lib/internal/streams/writable.js#L777-L782

`Duplex` doesn't have any success accessor so `Duplex#closed` is just undefined.

So, I believe this patch should be safe. (Whether this library should be using Node streams in general is a much bigger discussion.)